packages/core/src/contrib/Functions.ts

```diff
@@ -2810,6 +2865,47 @@ export const compDict = {
     },
     returns: valueT("Real2"),
   },
+  rayIntersectDistance: {
```

Comment:
Is it possible to expose different variants of `rayIntersectDistance` to the Style DSL as well? For example, aside from `rayIntersect`, we also have `rayIntersectCircle`, etc. Same goes for `closestSilhouetteDistance`. Otherwise there is no way for the Style writers to call `rayIntersectDistance` on abstract parameters ...

Reply:
Done.